src/updaters/generic.ts

```diff
@@ -51,6 +51,22 @@ type BlockScope = 'major' | 'minor' | 'patch' | 'version';
  * numbers
  */
 export class Generic extends DefaultUpdater {
+  private readonly inlineUpdateRegex: RegExp;
+  private readonly blockStartRegex: RegExp;
+  private readonly blockEndRegex: RegExp;
+
+  constructor(
+    options: UpdateOptions,
+    inlineUpdateRegex = INLINE_UPDATE_REGEX,
+    blockStartRegex = BLOCK_START_REGEX,
+    blockEndRegex = BLOCK_END_REGEX
```

Comment:
Can we roll these options into an extended options class (like `GenericUpdateOptions`)? If for whatever reason, we need to add more optional params, then the caller doesn't have to specify the earlier default values unless they're being overridden.